VM/rel2vm_params.py

```diff
@@ -347,7 +348,7 @@ def reduce_domain(
             close=False,
         )
         isections, icomps = gmt.intersections(
-            ["%s/srf.path" % wd, NZ_LAND_OUTLINE, "%s/tempEXT.tmp" % wd],
+            ["%s/srf.path" % wd, NZ_LAND_OUTLINE.as_posix(), "%s/tempEXT.tmp" % wd],
```

Comment:
If we are changing this, it should be an f-string.

Reply:
Sure done.